### PR TITLE
var_model flake8

### PR DESCRIPTION
**Pull request by jbrockmendel**
flake8 fixes, mostly whitespace-related

A handful of TODO notes for things that I had to restrain myself from doing in this PR.

The only non-aesthetic edit made here is deleting `forecast_cov` function, which was identical to `mse`.

The remaining style-based change I'd like to make -- but kept out of this PR because it will take more reviewer effort-- is getting rid of single-letter variables `p` and `k` and replacing them with their more informative counterparts `k_ar` and `neqs`.

### CODE REVIEW

File: statsmodels/tsa/vector_ar/var_model.py [hunk 1]
```diff
@@ -197,6 +195,10 @@ def mse(ma_coefs, sigma_u, steps):
 
     return forc_covs
 
+
+forecast_cov = mse
```
Comment (jbrockmendel) [inline statsmodels/tsa/vector_ar/var_model.py]: Sure.  Just pushed a commit that flips those.

**Comment by jbrockmendel** [comment]
@josef-pkt This should be good to go.

**Comment by jbrockmendel** [comment]
ping

**Comment by bashtage** [comment]
@josef-pkt LGTM - simple and useful clean up.  
@jbrockmendel Can the related PR be closed?

**Comment by jbrockmendel** [comment]
I think so, yes.

**Comment by josef-pkt** [comment]
rebases without problems, I will merge it later today

**Comment by josef-pkt** [comment]
merged on command line after rebase

merge commit message links to this PR (this time)
>git merge --no-ff -e var_flake8_3815_rebased

`-e` invokes editor for commit message
(default git merge message does not add the second line with commit code which github does)

Thanks @jbrockmendel @bashtage

**Pull request closed without merging**